src/compile/model.ts

```diff
@@ -469,13 +469,23 @@ export abstract class Model {
     return varName((this.name ? this.name + '_' : '') + text);
   }
 
+  public getDataName(type: DataSourceType) {
+    switch (type) {
```

Comment:
You can lookup the name on the enum, I think. Try `return DataSourceType [type]`.

Reply:
Thanks! I'm coming back to TypeScript after about 6 years away. I'm a little rusty still.

Reply:
It has changed a lot.